hack/lib/golang.sh

```diff
@@ -713,7 +719,7 @@ kube::golang::build_binaries() {
     fi
 
     local binaries
-    binaries=($(kube::golang::binaries_from_targets "${targets[@]}"))
+    while IFS="" read -r binary; do binaries+=("$binary"); done < <(kube::golang::binaries_from_targets "${targets[@]}")
```

Comment:
```
binaries=($(kube::golang::binaries_from_targets "${targets[@]}"))
          ^-- SC2207: Prefer mapfile or read -a to split command output (or quote to avoid splitting).
```